core/arch/arm/include/mm/tee_mmu_defs.h

```diff
@@ -32,7 +32,7 @@
 #define TEE_MMU_TTBCR_N_VALUE		7
 
 /* Number of sections in ttbr0 when user mapping activated */
-#define TEE_MMU_UL1_NUM_ENTRIES         ((1 << TEE_MMU_TTBCR_N_VALUE) / 4)
+#define TEE_MMU_UL1_NUM_ENTRIES         (1 << (12 - TEE_MMU_TTBCR_N_VALUE))
```

Comment:
This calculation is obviously not trivial, please add a comment explaining what's going on.